test/acceptance-tests/use-cases/test-scripts/create-model-config.js

```diff
@@ -48,6 +48,8 @@ module.exports = function() {
       if (err) return next(err);
       const config = data[testsuite.modelId];
       expect(config).to.not.to.be.undefined();
+      delete testsuite.ModelConfig.id;
+      delete testsuite.ModelConfig.facetName;
```

Comment:
We should remove the assertion on line 24, you should only be asserting in `this.Then`. The lines here probably need to be moved up into `this.Given` as this is part of your test setup.